cmd/openshift-tests/provider.go

```diff
@@ -61,6 +61,11 @@ func initializeTestFramework(context *e2e.TestContextType, config *exutilcluster
 
 	e2e.AfterReadingAllFlags(context)
 	context.DumpLogsOnFailure = true
+
+	context.IPFamily = "ipv4"
+	if config.HasIPv6 {
+		context.IPFamily = "ipv6"
+	}
```

Comment:
hm... I think I'd go with `if config.HasIPv6 && !config.HasIPv4`
also maybe add a comment pointing out that these strings come from kube e2e and are not equal to `corev1.IPv4Protocol` / `corev1.IPv6Protocol`